h2o-core/src/main/java/hex/AUC2.java

```diff
@@ -28,7 +28,7 @@
   public final double[] _tps;     // True  Positives
   public final double[] _fps;     // False Positives
   public final double _p, _n;     // Actual trues, falses
-  public final double _auc, _gini, _pr_auc; // Actual AUC value
```

Comment:
Is this still needed?

Reply:
Uses in tests heavily, would not dare to remove it in this PR.